conu/apidefs/backend.py

```diff
@@ -0,0 +1,53 @@
+"""
+Definition for a backend class
+"""
+
+import logging
+
+from .container import Container
+from .image import Image
+
+
+__version__ = "0.0.1-alpha"
+
+
+def set_logging(level=logging.INFO):
+    """
+    Set personal logger for this library.
+
+    :param level: int, see logging.{DEBUG,INFO,ERROR,...}
+    :return: logger instance
+    """
+    logger = logging.getLogger("conu")
+    # do we want to propagate to root logger?
+    # logger.propagate = False
+    logger.setLevel(level)
+
+    # logging to std.err
+    handler = logging.StreamHandler()
```

Comment:
What about if I would like to use FileHandler? I would prefer to create a function which will consume only handler.
Does it make sense to have FileHandler, at all?

Reply:
we can parametrize the function: this is a good suggestion, not sure if anyone is interested in logging to a file

Reply:
Good example is here: https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/logger.py

Reply:
see my comment above